catalyst/metrics/functional/_hitrate.py

```diff
@@ -5,6 +5,15 @@
 from catalyst.metrics.functional._misc import process_recsys_components
 
 
+def _nan_to_num(tensor, nan=0.0):
+    tensor = torch.where(torch.isnan(tensor), torch.ones_like(tensor) * nan, tensor)
+    return tensor
+
+
+# nan_to_num is available in PyTorch only from 1.8.0 version
+NAN_TO_NUM_FN = torch.__dict__.get("nan_to_num", _nan_to_num)
```

Comment:
In my opinion it will be better to check version of pytorch directly
```
if torch.__version__ >= "1.8.0":
    from torch import nan_to_num
else:
    define nan_to_num (tensor, nan=0.0):
    tensor = torch.where(torch.isnan(tensor), torch.ones_like(tensor) * nan, tensor)
    return tensor
```